src/renderer.rs

```diff
@@ -503,7 +503,7 @@ fn entry_row(
                             }
                         }@else {
                             a.directory href=(parametrized_link(&entry.link, sort_method, sort_order, raw)) {
-                                (entry.name) "/"
+                                "📁 " (entry.name) "/"
```

Comment:
I like this! However, could you please put this into the SCSS part rather than directly here? Take a look at how `symlink-symbol` is done.

Reply:
sure,  I will take a look there.